liberty-maven-plugin/src/it/dev-it/src/test/java/net/wasdev/wlp/test/dev/it/DevInvokeTestingTest.java

```diff
@@ -0,0 +1,133 @@
+/*******************************************************************************
+ * (c) Copyright IBM Corporation 2019.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package net.wasdev.wlp.test.dev.it;
+
+import static junit.framework.Assert.assertTrue;
+import static junit.framework.Assert.assertFalse;
+
+import java.io.File;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class DevInvokeTestingTest extends BaseDevTest {
+
+   static File testResources = new File("../resources");
+
+   @BeforeClass
+   public static void setUpBeforeClass() throws Exception {
+      setUpBeforeClass(null);
```

Comment:
Perhaps line 36 should call `BaseDevTest.setUpBeforeClass(null)` to stay consistent with line 41, alternatively line 41 could be changed.

Reply:
Changed to `BaseDevTest.setUpBeforeClass()`